hswindow/hstoolwindow/hsmpqexplorer/mpqexplorer.go

```diff
@@ -117,7 +117,7 @@ func (m *MPQExplorer) GetMpqTreeNodes() []g.Widget {
 		go func(idx int) {
 			mpq, err := d2mpq.FromFile(filepath.Join(m.config.AuxiliaryMpqPath, m.project.AuxiliaryMPQs[idx]))
 			if err != nil {
-				log.Fatal("failed to load mpq: ", err)
+				log.Print("failed to load mpq: ", err)
```

Comment:
`mpq` may be nil, so I suggest to return after `log.Print` call

Reply:
this is inside of a go func. I fixed by also checking if mpg is nil before using it.